services/movie.py

```diff
@@ -0,0 +1,38 @@
+from django.db.models import QuerySet
+
+from db.models import Movie
+
+
+def get_movies(genres_ids: list[int] = None,
+               actors_ids: list[int] = None) -> QuerySet:
+    query_set_movies = Movie.objects.all()
+
+    if genres_ids and actors_ids:
+        query_set_movies = query_set_movies.filter(genres__id__in=genres_ids,
+                                                   actors__id__in=actors_ids)
```

Comment:
The current filtering logic requires movies to match both genre and actor IDs when both are provided. If the intention is to filter by either condition, consider using `Q` objects to combine filters with an OR condition.